source/gui/settingsDialogs.py

```diff
@@ -1122,24 +1122,23 @@ def postInit(self):
 		self.synthList.SetFocus()
 
 	def updateSynthesizerList(self):
-		driverList=getSynthList()
-		self.synthNames=[x[0] for x in driverList]

```

Comment:
this should not be removed, this is an API breaking change.
I'd encourage reverting these unrelated changes.